src/diffusers/pipelines/pipeline_loading_utils.py

```diff
@@ -70,6 +70,8 @@
         "SchedulerMixin": ["save_pretrained", "from_pretrained"],
         "DiffusionPipeline": ["save_pretrained", "from_pretrained"],
         "OnnxRuntimeModel": ["save_pretrained", "from_pretrained"],
+        "ChatGLMTokenizer": ["save_pretrained", "from_pretrained"],
+        "ChatGLMModel": ["save_pretrained", "from_pretrained"],
```

Comment:
@yiyixuxu  do we have an alternative? Couldn't find any other pipeline that have their own tokenizer. I can use the `ModelMixin` for the `ChatGLModel` to avoid adding it here but not for the tokenizer.

Reply:
oh I don't think it is needed here, they are technically "transfomers" classes because `ChatGLMModel` inherits from `PreTrainedModel` and `ChatGLMTokenizer` inherits from `PreTrainedTokenizer`

Reply:
yeah I tried that, but if I change it to transformers, since it has custom code it doesn't work (NotImplementedError), so in the model config I have to put:

```
{
  ...
  "text_encoder": [
    "diffusers",
    "ChatGLMModel"
  ],
  "tokenizer": [
    "diffusers",
    "ChatGLMTokenizer"
  ],
  ...
}
```

with that config in the model, even though they inherit from `PreTrainedModel` and `PreTrainedTokenizer`, the loading doesn't work.

Reply:
so like here https://huggingface.co/kandinsky-community/kandinsky-2-1/blob/main/model_index.json
try this?
```python
  "text_encoder": [
    "kolors",
    "ChatGLMModel"
  ],
```

Reply:
oh yeah, I missed that, thanks.